client/my-sites/themes/themes-upload-card/index.jsx

```diff
@@ -35,13 +35,15 @@ class ThemeUploadCard extends React.Component {
 					label={ this.props.label }
 					count={ this.props.count }
 				>
-					<Button compact icon
-						onClick={ this.trackClick }
-						href={ this.props.href }
-					>
-						<Gridicon icon="cloud-upload" />
-						{ translate( 'Upload Theme' ) }
-					</Button>
+					{ this.props.href &&
```

Comment:
Okay, after some discussion on Slack, it's probably better to leave it in after all, since it contains the site slug which we would otherwise have to fetch from Redux state.